CONTRIBUTING.md

```diff
@@ -1 +1 @@
-All changes are welcome as long as no code is involved. If you run into any bugs, please file an issue and explain how that was even possible.
+All changes are welcome, as long as no code is involved. If you run into any bugs, please file an issue and explain how that was even possible.
```

Comment:
The comma is redundant, this is implying that the dependent clause is separate from the independent clause.